---
title: "'Catch' 블록에 도달할 수 없습니다 <exception> 동일한 'Try' 문의 윗부분에서 처리"
ms.date: 07/20/2015
f1_keywords:
- bc42031
- vbc42031
helpviewer_keywords:
- BC42031
ms.assetid: 7d15597c-5988-42ea-a853-63cbf78faaf3
ms.openlocfilehash: e05a324e67f279c650626833a6745f28df388c0d
ms.sourcegitcommit: 2701302a99cafbe0d86d53d540eb0fa7e9b46b36
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 04/28/2019
ms.locfileid: "64627197"
---
# <a name="catch-block-never-reached-exception-handled-above-in-the-same-try-statement"></a>'Catch' 블록에 도달할 수 없습니다 \<예외 > 동일한 'Try' 문의 윗부분에서 처리
코드의 `Catch` 블록이 앞의 `Try` 블록에서 처리되므로 도달할 수 없습니다.  
  
이 메시지는 기본적으로 경고입니다. 경고를 숨기거나 오류로 처리하는 방법에 대한 자세한 내용은 [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic)을 참조하세요.
  
 **오류 ID:** BC42031  
  
## <a name="to-correct-this-error"></a>이 오류를 해결하려면  
  
- 중복 문을 제거합니다.  
  
## <a name="see-also"></a>참고자료

- [Try/catch 블록을 사용 하 여 예외를 catch 하는 방법](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [사용자 필터 예외 처리기 사용](../../standard/exceptions/using-user-filtered-exception-handlers.md)
- [Try...Catch...Finally 문](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
